### PR TITLE
Move viewer keybindings

### PR DESCRIPTION
**Pull request by sofroniewn**
# Description
I'm looking into moving our keybindings off of the `Viewer` and onto the `ViewerModel` as much as possible, this would match the `Layer` objects where the keybindings are. I think in the long run we probably want to allow for importing all our objects without keybindings attached, but for now this will further the objectives of #1808 by bringing our Viewer and ViewerModel closer together.

I'm running into a problem though which appears to be the the same as reported in #1535 which is that the keybindings don't work when inherited. We probably want to fix that in a standalone PR, but raising this now so that we have another example - cc @jni @kne42 

## Type of change

- [x] Refactor (non-breaking change which fixes an issue)

### CODE REVIEW

File: napari/__init__.py [hunk 1]
```diff
@@ -57,7 +57,7 @@
 # Note that importing _viewer_key_bindings is needed as the Viewer gets
 # decorated with keybindings during that process, but it is not directly needed
 # by our users and so is deleted below
-from . import _viewer_key_bindings  # noqa: F401
+# from . import _viewer_key_bindings  # noqa: F401
```
Comment (sofroniewn) [inline napari/__init__.py]: Note for debugging I'm not loading the `Viewer` keybindings, only the `ViewerModel` ones should be functional, but they aren't - same as was reported in #1535

**Comment by sofroniewn** [comment]
I've updated this PR to also fix a regression from #1434 which where we lost the "highlight" coloring of our active dims slider. In theory that could be its own PR and I could pull that out if this PR becomes controversial, but otherwise it's in here for now.

I've moved the fullscreen and toggle play keybindings to the window. All other keybindings are on the `ViewerModel` and there are no more keybindings on the `Viewer` object itself.

I likely still have some tests to fix, and note that this won't work until #1535 is fixed

**Comment by sofroniewn** [comment]
Ok, I have merged the changes from #1838, this is now ready for review @jni. Note that it also includes a fix for the last used slider highlighting.

**Review by jni — APPROVED** [review]
Awesome! It's a long PR but so simple in the end! :tada:

**Pull request merged**